v1.1.1/js/Loopy.js

```diff
@@ -77,6 +82,8 @@ function Loopy(config){
 		if(!self.modal.isShowing){ // modAl
 			self.model.update(); // modEl
 		}
+		window.addEventListener("wheel", self.zoom);
```

Comment:
Think we could bind `-` and `=` to be zoom out and zoom in respectively as well? This works on a laptop trackpad, but it's kinda unwieldy. The ability to take predetermined step size zooms would be helpful.

Reply:
Good idea I'll try to implement that

Reply:
Coolio, thanks!